src/infrastructure/TransactionService.js

```diff
@@ -176,8 +176,9 @@ class TransactionService {
       pageNumber,
       pageSize,
       orderBy: 'desc',
-      transactionTypes: filterVaule === '0' ? [] : [filterVaule],
-      group: 'Confirmed'
+      transactionTypes: [],
+      group: 'Confirmed',
```

Comment:
i think you can remove 
`transactionTypes` and `group`
because i saw you already pass `transactionTypes` from store.

Reply:
It is default values. For example, when we pass some transactionTypes in the Store, we should not add `gruop: 'Confirmed'` to each option.